src/presentation/pages/NoteSettings.vue

```diff
@@ -56,8 +56,9 @@ const invitationLink = computed(
   () => `${import.meta.env.VITE_PRODUCTION_HOSTNAME}/join/${noteSettings.value?.invitationHash}`
 );
 
-load(props.id);
-
+onMounted(() => {
```

Comment:
describe why this change has been added

Reply:
still actual. Added doc is useless, it just repeats the code. It should describe why this code has been added?

Reply:
still ignored